server/views/pages/accessDenied.njk

```diff
@@ -0,0 +1,33 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - access denied" %}
+{% set pageId = "access-denied-page" %}
+
+{% block content %}
+    <div class="govuk-grid-row govuk-body">
```

Comment:
The `govuk-body` class should be used on each `<p>` tag, not to worry about these html comments though, they don't make any difference

Reply:
I'll make a note of these things to check and sort as a follow on PR along with access denied page content.